_episodes/05-starting-jekyll.md

```diff
@@ -21,4 +21,57 @@ keypoints:
 - use it again in another page
 - change the value of the global parameter and study the effect
 
+
+> ## Creating your own variable
+>
+> Let's practice making and using variables.
+> Think of a variable you may want to use in your `index.md` page.
+> Some possible options include: your name, the institution you work at, the PI of your lab group.
+> Create the variable and reference it in your `index.md` and save/commit that to your repository.
+> What did you add to your `index.md` to create this variable?
+> Where did you add the line above to your `index.md`?
+> How did you reference that variable?
+>
+> > ## Solution
+> >
+> > We will create the `pi` variable by adding the following line to the yaml header.
+> > ~~~
+> > pi: "Ada Lovelace"
+> > ~~~
+> >
+> > Here is an example of using it in the `index.md` file
+> > ~~~
+> > This project is lead by {{ page.pi }}.
+> > ~~~
+> > {: .output}
+> {: .solution}
+{: .challenge}
+
+
+> ## Making a global variable
+>
+> Grad student Davis wants to use their current institution name throughout their personal website
+> but will soon be leaving to become a postdoc at another institution.
+> They have created a variable `current_institution` in the YAML header of the `index.md` pages on their site.
+> What changes would they make to use this variable on other pages as well?
+> How might they need to change the following text on their `index.md` page after those changes?
+>
+> ~~~
+> Davis currently works at {{ current_institution }}.
```

Comment:
```suggestion
> Davis currently works at {% raw %}{{ page.current_institution }}{% endraw %}.
```
wrapping the curlies in `{% raw %}` tags so the page doesn't try to substitute in a non-existent variable name. Also adding `page.` before the variable name.